hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -441,6 +440,7 @@ private enum State {
   private static final int MSECS_PER_MINUTE = 60 * 1000;
 
   private final boolean isSecurityEnabled;
+  private ScheduledExecutorService executorService;
```

Comment:
I believe this should be named as certificateExpirationMonitor or something similar.